sklearn/tests/test_common.py

```diff
@@ -273,6 +280,44 @@ def test_strict_mode_parametrize_with_checks(estimator, check):
     check(estimator)
 
 
+def _generate_search_cv_instances():
+    for SearchCV, (Estimator, param_grid) in zip(
+        [GridSearchCV, RandomizedSearchCV],
+        [
+            (Ridge, {"alpha": [0.1, 1.0]}),
+            (LogisticRegression, {"C": [0.1, 1.0]}),
+        ],
+    ):
+        yield SearchCV(Estimator(), param_grid)
+
+    for SearchCV, (Estimator, param_grid) in zip(
+        [GridSearchCV, RandomizedSearchCV],
```

Comment:
What about `HalvingGridSearchCV` and `HalvingRandomSearchCV`?

Reply:
It might get tricky to get the min and max resources on the small toy datasets that we use for common testing, so maybe we should ignore those and assume that testing for base GS and RS is enough.

Reply:
It was not there at the time. But we could investigate what is needed in another PR.